bio_hansel/subtyper.py

```diff
@@ -473,6 +474,7 @@ def subtype_reads_ac(reads: Union[str, List[str]],
     st.non_present_subtypes = [x for x in possible_downstream_subtypes
                                if not (df.subtype == x).any()]
     st.qc_status, st.qc_message = perform_quality_check(st, df, subtyping_params)
+    st.coverage = "{0:.3f}".format(df['freq'].mean())
```

Comment:
Why not leave as numeric? When writing the results to a CSV/tab-delimited file with Pandas, you can specify the float format:

```python
df.to_csv('out.csv', float_format='%.3f')
```

Reply:
Oh perfect, I didn't know you could do that. I'll change it back to numeric.